LEAF_Request_Portal/templates/reports/LEAF_inbox_combined.tpl

```diff
@@ -125,13 +129,40 @@
                      $('#inboxForm'+ hash +'_'+depID+'_'+data.recordID).slideDown();
                      $('#requestTitle').attr('tabindex', '0');
                      $('#requestInfo').attr('tabindex', '0');
-                     ariaSubIndicators(1);
                     }
                 })
             }
             })
             }
             },
+            {name: 'Status', indicatorID: 'currentStatus', editable: false, callback: function(data, blob) {
```

Comment:
Need variables to be properly scoped.  These should probably be "let" scoping?

Reply:
The variable scope has been changed to "let" as requested.  Will keep an eye on this in future JS changes as well.